### DISCUSSION TITLE
Is there "Delete Conversation" endpoint?

### DISCUSSION THREAD

**Discussion by emreond** [sbertix/Swiftagram#234] [Q&A] [answered]
Hello,

Is it possible to delete conversation via endpoint? As I see we can fetch conversations as a list but can't find any endpoint for removing them.

**Comment by sbertix** [comment, marked as answer]
Hey @emreond, 

The endpoint for deleting conversations is not available in **Swiftagram**, and it'll probably never be. 
As a rule of thumb, undoable destructive actions are not added to the library, in order to prevent misuse.
